target/linux/lantiq/base-files/lib/preinit/05_set_preinit_iface_lantiq

```diff
@@ -10,6 +10,12 @@ set_preinit_iface() {
 	TDW8970)
 		ifname=eth0
 		;;
+	TDW8980)
```

Comment:
Do not add a branch which set the same value as an existing branch. Use ```TDW8970|TDW8980)``` instead.